content/articles/picking-pdf-and-image-from-phone-storage/index.md

```diff
@@ -0,0 +1,322 @@
+This tutorial is going to cover on how to open gallery and files apps from your android application and pick an image and a PDF file. You will learn about intents specifically implicit intents. 
+There has been a challenge of choosing files from the phone storage since the contents available on the internet are ambiguous. This tutorial is going to simplify your work. 
+
+### Table of Contents
+- [Prerequisites](#prerequisites)
+- [Goals](#goals)
+- [What is Intents](#what-is-intents)
+- [Types of Intents](#types-of-intents)
+- [Adding Permissions](#adding-permissions)
+- [Designing Layout](#designing-layout)
+- [Picking Image](#picking-image)
+- [Picking PDF file](#picking-pdf-file)
+- [Coclusion](#conclusion)
+
+### Prerequisites
+Understanding this tutorial requires that you have: 
+- [Android Studio IDE](https://developer.android.com/studio/index.html) installed.
+- Basic understanding of [Kotlin](https://kotlinlang.org/) programming language. 
+- Basic understanding of [Intents](https://developer.android.com/guide/components/intents-filters).
+
+### Goals
+- Picking images from gallery and loading to `ImageView`.
+- Taking photos and loading to `ImageView`.
+- Picking PDF file and display in a `TextView`. 
+
+File choosing in Android involves implicit intents. Implicit intent is a type of intent that navigates the user to another application. For File choosing to be successful, certain permissions MUST  be allowed in the app manifest.
+
+### What is Intents
+Is an object that facilitates communication between app components. Intents are used in cases when an application wants to start an activity, start a service, or when delivering a broadcast.
+
+### Types of Intents
+Intents consist of two types:
+
+#### Implicit intents
+Is a type of intent that facilitates communication between two different apps. For instance, in this tutorial, we are going to imply implicit intent to communicate with the gallery and files apps. Implicit intents only need the declaration of the general action to be performed.
+
+#### Explicit intents
+Is a type of intent that allows communication within an application itself. For example, when you want to navigate from one activity to the next, you will trigger an explicit intent. 
+
```

Comment:
Change to "
An explicit intent is a type of intent that allows communication between application components. For example, when you want to navigate one activity to the next, you will trigger an explicit intent. "